Week2/homework/js-exercises/Exercise5.js

```diff
@@ -0,0 +1,14 @@
+'use strict';
+
+let drinkTray =[];
+
+const drinkTypes = ["cola", "lemonade", "water"];
+
+let index = 0;
+while(index < 5){
+
+    drinkTray.push(drinkTypes[ index % drinkTypes.length]);
+    index++;
+
+}
+console.log("Hey guys, I brought a " + drinkTray.sort());
```

Comment:
Nicely solved, maybe a good challenge would be to handle "random" picks from the drinks. Meaning you don't know in what order the drinks will be selected.